packages/components/src/components/tooltip/tooltip.tsx

```diff
@@ -9,7 +9,7 @@ type TTooltip = {
     classNameIcon?: string;
     has_error?: boolean;
     icon?: string;
-    message?: string;
+    message?: string | null;
```

Comment:
why did you change this? message is optional param. if we want to reset somehow the message we can set it to empty line, rather than add new type

Reply:
@yauheni-deriv, I changed this, because they explicitly pass in `null` to this component in the `input-field.tsx` file as seen in the screenshot

<img width="879" alt="null" src="https://user-images.githubusercontent.com/104334373/220887296-4b926124-0530-4edd-ac65-3e8bc8d132ea.png">

Reply:
we can pass here empty line then, no?
<img width="999" alt="Screenshot 2023-02-23 at 14 20 23" src="https://user-images.githubusercontent.com/103182683/220892091-0be5cf2f-7bc4-499d-ac4c-d182cb9a6e35.png">
here if this message is null, data will be undefined anyway. because null is falsy value, as well as empty line.
so no needs to add more types.
how do you think?

Reply:
Okay @yauheni-deriv sure. I'll check this later